package.json

```diff
@@ -6,18 +6,19 @@
   "license": "MIT",
   "scripts": {
     "analyze": "yarn workspace patternfly-org-4 analyze",
-    "develop": "yarn develop:v4",
-    "develop:v4": "yarn workspace patternfly-org-4 develop",
-    "build": "yarn build:v4",
-    "build:analyze": "yarn workspace patternfly-org-4 build:analyze && yarn copy:v4",
-    "build:v4": "yarn workspace patternfly-org-4 build && yarn copy:v4",
-    "copy:v4": "rm -rf build/patternfly-org/v4 && mkdir -p build/patternfly-org && cp -r packages/v4/public build/patternfly-org/v4 && cp -r build/patternfly-org/v4/assets build/patternfly-org/assets",
+    "develop": "yarn develop:v5",
+    "develop:v5": "yarn workspace patternfly-org-4 develop",
```

Comment:
Can we update `packages/documentation-site/package.json` name & description from `patternfly-org-4`, and update the references in this file?  Would this break anything?

Reply:
I think it should be fine but may break the GH CI build again if the workflow files still targets it.